_posts/2024-01-19-Interspeech2024-Discrete-Speech-Unit-Challenge.md

```diff
@@ -229,6 +322,15 @@ Papers for the Interspeech Special Session have to be submitted following the sa
 
 Please use the [submission URL](https://cmt3.research.microsoft.com/INTERSPEECH2024)) to submit papers and select "14.10 Speech Processing Using Discrete Speech Units (Special Session)" as the primary subject areas for your paper submission.
 
+##  Submission pakcage submission
+
+Participants will need to submit their submission package through the [google form](https://forms.gle/XAdQ5WyEVD2tDwYh8), where the leaderboard will be updated accordingly within 48 hours. The final ranking information will be recorded at the time of Interspeech submission deadline (i.e., 2024/03/11 AOE).
```

Comment:
Can you give me an access to this form?

We should make this part very careful and structured.

Reply:
Sure, send you the access